doc-deploy-changelog/action.yml

```diff
@@ -73,21 +73,55 @@ inputs:
     default: true
     type: boolean
 
-  update-release-branch:
+  release-from-main:
     description: |
-      Whether or not to update the CHANGELOG in the release branch.
-      If ``true``, the branch used to update the CHANGELOG is "release/major.minor".
-      If ``false``, the branch used to update the CHANGELOG is
-      "maint/changelog-update-tag_version" and will be deleted after use.
-    required: false
-    default: true
-    type: boolean
+      Pushing a release tag results in this action running twice:
```

Comment:
@klmcadams you will have to try building the docs locally and identify what the issue is

Reply:
@RobPasMue the issue seemed to be the dashes in the bullet list, so I removed them. Is that okay?

Reply:
Yep all good - you could've used asterisks though